Description: Fix indentation to avoid compiler warning
Author: Jeffrey Ratcliffe <jjr@debian.org>
Last-Update: 2017-06-22
--- a/src/gtkimagenav.c
+++ b/src/gtkimagenav.c
@@ -71,8 +71,8 @@
 gtk_image_nav_get_preview_size (GtkImageNav *nav)
 {
 	GdkPixbuf *pixbuf = gtk_image_view_get_pixbuf (nav->view);
-    if (!pixbuf)
-        return (Size){GTK_IMAGE_NAV_MAX_WIDTH, GTK_IMAGE_NAV_MAX_HEIGHT};
+	if (!pixbuf)
+		return (Size){GTK_IMAGE_NAV_MAX_WIDTH, GTK_IMAGE_NAV_MAX_HEIGHT};
 	int img_width = gdk_pixbuf_get_width (pixbuf);
 	int img_height = gdk_pixbuf_get_height (pixbuf);
 
